src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import MovieCard from '../Movie-Card/MovieCard';
+import { Route, NavLink } from 'react-router-dom';
+import './SearchBar.css';
+
+class SearchBar extends Component {
+  constructor({ movieData, handleClick }) {
+    super();
+      this.state = {
+        movies: movieData,
+        matchingMovies: null,
+        searchInput: '',
+        error: ''
+      }
+  }
+
+  handleChange = (event) => {
+    event.preventDefault();
+    let searchValue = (event.target.value).toUpperCase();
+    this.setState({ searchInput: searchValue })
```

Comment:
This is all wonderfully put together. I think that just below this line we could connect from our app.js via props coming through our constructor and super.